documentation/source/simulator_support.rst

```diff
@@ -129,7 +142,21 @@ For Verilator 4.102 and above, the `-CFLAGS -DVM_TRACE_FST=1` argument is no lon
 
     EXTRA_ARGS += --trace-fst --trace-structs -CFLAGS -DVM_TRACE_FST=1
 
-The resulting file will be ``dump.fst`` and can be opened by ``gtkwave dump.fst``.
+The resulting file will be :file:`dump.fst` and can be opened by :file:`gtkwave dump.fst`.
+
+.. _sim-verilator-existing-setups:
+
+Extending existing Simulation Setups
+------------------------------------
+
+In order to extend an existing simulation setup for use with cocotb,
+this section shows the minimum settings to be done.
+
+* Define the :envvar:`LIBPYTHON_LOC` environment variable using ``$(cocotb-config --libpython)``.
+* Define the :envvar:`MODULE` environment variable with the name of the Python module containing your testcases.
+* Extend the call to ``verilator`` with these options:
+  ``-cc --exe --vpi --public-flat-rw --prefix Vtop -LDFLAGS "-Wl,-rpath,$(shell cocotb-config --lib-dir) -L$(shell cocotb-config --lib-dir) -lcocotbvpi_verilator -lgpi -lcocotb -lgpilog -lcocotbutils" $(cocotb-config --share)/lib/verilator/verilator.cpp``
```

Comment:
This looks strange.

Reply:
Indeed, of the part before `-LDFLAGS` I only need the `--vpi` option.

Reply:
Like it is renders very very long need to break in 2 lines.